app/views/measures/_measures.html.erb

```diff
@@ -50,18 +50,33 @@
   <!-- Export tab -->
   <div class="govuk-tabs__panel" id="export">
     <h2 class="govuk-heading-l"><%= measures_heading(anchor: 'export') %></h2>
-    <p>The commodity code for exporting and <%= link_to 'Intrastat reporting', 'https://www.gov.uk/intrastat', target: "_blank" %> is <%= declarable.display_export_code %></p>
+    <p>The commodity code for exporting and <%= link_to 'Intrastat reporting', 'https://www.gov.uk/intrastat', target: "_blank" %> is <%= declarable.display_export_code %>.</p>
 
     <%= render partial: 'declarables/consigned', locals: { declarable: declarable } %>
 
-    <% if declarable.export_measures.any? %>
+    <% if uk_declarable.presence&.export_measures&.any? %>
```

Comment:
I think we'll want to update https://github.com/trade-tariff/trade-tariff-frontend/blob/main/app/views/measures/_measures.html.erb#L145-L154 just to make sure the measure modals work as before